src/gsy_e/gsy_e_core/sim_results/file_export_endpoints.py

```diff
@@ -244,64 +229,86 @@ def _specific_row(self, slot, market):
-            return [
-                round(
-                    self.area.strategy.state.tanks.get_unmatched_demand_kWh(slot), ROUND_TOLERANCE
-                ),
-                round(self.area.strategy.state.get_storage_temp_C(slot), ROUND_TOLERANCE),
-                round(self.area.strategy.state.get_cop(slot), ROUND_TOLERANCE),
-                round(self.area.strategy.state.get_heat_demand(slot), ROUND_TOLERANCE),
-                round(self.area.strategy.state.get_condenser_temp(slot), ROUND_TOLERANCE),
-            ]
-        # pylint: disable=unidiomatic-typecheck
-        if type(self.area.strategy) == MultipleTankHeatPumpStrategy:
-            cop = self.area.strategy.state.heatpump.get_cop(slot)
-            return [
-                round(
-                    self.area.strategy.state.charger.tanks.get_average_tank_temperature(slot),
-                    ROUND_TOLERANCE,
-                ),
                 round(
-                    self.area.strategy.state.charger.tanks.get_min_heat_energy_consumption_kJ(
-                        slot
-                    ),
-                    ROUND_TOLERANCE,
+                    self.area.strategy.state.heatpump.get_heat_demand(slot) / 1000, ROUND_TOLERANCE
                 ),
-                round(cop, ROUND_TOLERANCE),
-                round(self.area.strategy.state.heatpump.get_heat_demand(slot), ROUND_TOLERANCE),
-            ]
-        # pylint: disable=unidiomatic-typecheck
-        if type(self.area.strategy) == MultipleTankVirtualHeatpumpStrategy:
-            cop = self.area.strategy.state.heatpump.get_cop(slot)
-            return [
-                round(
-                    self.area.strategy.state.tanks.get_unmatched_demand_kWh(slot), ROUND_TOLERANCE
-                ),
-                round(
-                    self.area.strategy.state.tanks.get_average_tank_temperature(slot),
-                    ROUND_TOLERANCE,
-                ),
-                round(
-                    self.area.strategy.state.tanks.get_min_energy_consumption(cop, slot),
-                    ROUND_TOLERANCE,
-                ),
-                round(cop, ROUND_TOLERANCE),
-                round(self.area.strategy.state.heatpump.get_condenser_temp(slot), ROUND_TOLERANCE),
             ]
+        if file_key == "water_tanks":
+            for tank in self.area.strategy.state.charger.tanks.tanks_states:
+                if isinstance(tank, WaterTankState):
+                    rows.append(round(tank.current_tank_temperature(slot), ROUND_TOLERANCE))
+                    rows.append(round(tank.get_soc(slot), ROUND_TOLERANCE))
+        if file_key == "pcm_tanks":
+            for tank in self.area.strategy.state.charger.tanks.tanks_states:
+                if isinstance(tank, PCMTankState):
+                    rows.append(round(tank.current_tank_temperature(slot), ROUND_TOLERANCE))
+                    rows.append(round(tank.get_soc(slot), ROUND_TOLERANCE))
+        return rows
```

Comment:
Do you plan to change this in the future and report different information for PCM tanks and Water tanks? Because if not, then there is no need for having 2 separate files for the water and pcm tanks, one file should suffice for both.

Reply:
I wanted to separate the two very different storages. Also I wanted to make sure that the number of columns does not grow that much. Imagine, with the current implementation, if you have 4 water tanks, you have 9 columns in the CSV file which gets unhandy.
Main reason is the one you mentioned though. We might export different things for the PCM tank than the water tank, even in this study already.